data/mods/Medieval_Stuff/armor.json

```diff
@@ -58,7 +58,7 @@
         "encumbrance" : 10,
         "bashing" : 10,
         "flags" : ["VARSIZE", "STURDY"],
-        "coverage" : 80,
+        "coverage" : 60,
```

Comment:
70% is horrible for a dedicated armor. No dedicated armor should be below 75%, not even the low tier ones.
Change encumbrance or something like that. The helmet covers eyes, so encumbrance matters here.